contrib/pg_stat_statements/expected/disable_pg_stat_statements.out

```diff
@@ -0,0 +1 @@
+
```

Comment:
Is it possible to organize the testing without having these empty files?

Reply:
Unfortunately not, if we intend to use regression tests for this, they require an output file present even if it is empty. It's maybe possible to use Makefile rules for this, but I think explicit tests are more readable.